python_scripts/cross_validation_train_test.py

```diff
@@ -258,6 +258,10 @@
 _ = plt.title("Test error distribution")
 
 # %% [markdown]
+# The argument `density=True` in the plot above normalizes the histogram such
```

Comment:
To be honest I don't think `density=True` is super useful either, but maybe that's just me ... it would be fine to show counts instead.

IMO `density=True` can lead to confusing y values since it is sensitive to the bin width.

Reply:
I was also thinking about avoiding this everywhere in the MOOC, mostly because we never really give a probabilistic interpretation to areas.

Reply:
I removed the `density=True` argument in all the histogram plots, what do you think?

Reply:
I am fine with this